submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/metrics/metrics.component.html

```diff
@@ -17,4 +17,25 @@
   ~ under the License.
   -->
 
-<p>metrics works!</p>
+<div id="metricTable">
+  <nz-table #metricTable nzSize="middle" [nzBordered]="true" [nzFrontPagination]="false" [nzData]="podMetric">
+    <thead>
+      <tr>
+        <th>Key</th>
+        <th>Value</th>
+        <th>Time</th>
+        <th>Step</th>
+        <th>Is_NaN</th>
```

Comment:
We could remove this line
If `Is_NaN` is true, we set metric value string to "Nan"